packages/config/config/devices/0x010f/fgd212_3.5.json

```diff
@@ -787,11 +713,11 @@
 			]
 		},
 		"47": {
-			"label": "OVERCURRENT alarm report",
-			"description": "(short circuit, burnt out bulb causing overcurrent)",
+			"label": "OVERCURRENT Alarm Report",
```

Comment:
Again, lose the all caps.

Reply:
Done in all places, I thought that OVERCURRENT etc might be some Z-Wave term that was supposed to be written like this.